src/parser/nmodl.yy

```diff
@@ -707,6 +707,10 @@ number          :   NUMBER
                         $2->negate();
                         $$ = $2;
                     }
+                |   "+" NUMBER
```

Comment:
@nrnhines : out of curiosity : for any reason this rule wasn't there in original implementation?